roles/gw/tasks/main.yml

```diff
@@ -68,7 +68,7 @@
   template:
     src: conntrackd.conf
     dest: /etc/conntrackd/conntrackd.conf
-    mode: "600"
+    mode: "644"
```

Comment:
now we're running conntrackd as root under sudo from nrpe, does this still need to change?

Reply:
oh! the script does its own parsing of the config file while still running as nagios, ignore me then

Reply:
I see your point that we could add both the HashLimit lookup and `conntrackd -s` under sudo rather than changing permissions for the former and using sudo for the latter.  However, I don't think we have any other choice but sudo for the latter.  So I think that was why it was written as it was, to avoid having more than necessary to sudoers.